simple.rb

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env ruby
+counts = Hash.new(0)
```

Comment:
This is good stuff! Using `Hash.new(0)` would remove a null check in my version! 👍